### PR TITLE
Update to Svelte 5

### PR DESCRIPTION
**Pull request by hnasheralneam**

### CODE REVIEW

**Comment by hnasheralneam** [comment]
@leowilkin

File: src/routes/RotatingSphere.svelte [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <script>
-  export let verticalLines = 10;
-  export let horizontalLines = 10;
-  export let style;
+  /** @type {{verticalLines?: number, horizontalLines?: number, style: any}} */
```
Comment (skyfallwastaken) [inline src/routes/RotatingSphere.svelte]:
```suggestion
  /** @type {{verticalLines?: number, horizontalLines?: number, style: string}} */
```

File: src/lib/Page.svelte [hunk 2]
```diff
@@ -1,13 +1,14 @@
 <script>
-  export let title;
+  /** @type {{title: any, children?: import('svelte').Snippet}} */
```
Comment (skyfallwastaken) [inline src/lib/Page.svelte]:
```suggestion
  /** @type {{title: string, children?: import('svelte').Snippet}} */
```

**Review by skyfallwastaken — CHANGES_REQUESTED** [review]
two small nitpicks here :)

**Pull request merged**